Sources/Orbit/Support/Forms/InputStyle.swift

```diff
@@ -91,14 +93,17 @@ struct InputContent<Content: View>: View {
     private var prefixColor: Color {
         switch (value, state) {
             case (_, .disabled):        return .cloudDarkerActive
-            case (.none, _):            return .inkNormal
             case (_, .modified):        return .blueDark
             default:                    return .inkNormal
         }
     }
 
     private var suffixColor: Color {
-        prefixColor
+        switch (value, state) {
+            case (_, .disabled):        return .cloudDarkerActive
+            case (_, .modified):        return .blueDark
+            default:                    return .inkLight
```

Comment:
I suppose `value` can be removed from the switch now..

Reply:
True, updated for both properties